src/components/App.css

```diff
@@ -1 +1,13 @@
-@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100&family=Roboto:wght@100;400;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;700&family=Roboto+Slab&display=swap');
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Roboto Slab', serif;
+  background-color: rgb(243, 243, 243);
+}
+
+.montserrat {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.813rem;
+}
```

Comment:
- [ ] The app layout should match the Zeplin design, kindly make sure that all of your elements have a similar padding/margins.
Zeplin Design
![image](https://user-images.githubusercontent.com/33767951/142711703-c8772ee7-dc59-46af-b67a-8c0360c3f92e.png)
Your Styling
![image](https://user-images.githubusercontent.com/33767951/142711711-b3c6420b-94e0-416d-9540-27b0ba29b3cf.png)